frontend/views/components/CreateGroupSteps/GroupName.vue

```diff
@@ -28,7 +28,7 @@ div
     ref='name'
     type='text'
     name='groupName'
-    :class="{ 'error': v.groupName.$error }"
+    :class='{ error: v.groupName.$error }'
```

Comment:
Hmm... @sandrina-p, seems we found out where the `v` from the previous PR is being used? Is this right? Or should be it `$v` or something from vuelidate? Can you investigate this?

Reply:
I can and I will check it when implementing the Remove Member :)